### PR TITLE
[ltio] a c++ implemented net-io based on coroutine and refactor

### PR DESCRIPTION
**Pull request by echoface**
a new test player from ltio.
please help review&merge
image

### CODE REVIEW

**Comment by nbrady-techempower** [comment]
The code for the actual tests must be included in this PR. The can't be pulled from outside sources.

**Comment by echoface** [comment]
> The code for the actual tests must be included in this PR. The can't be pulled from outside sources.

thanks for your support, pull request has updated.

File: frameworks/C++/ltio/http_benchmark.cc [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        response->SetKeepAlive(req->IsKeepAlive());
+
+        response->InsertHeader("Server", "ltio");
+        auto tm = fmt::gmtime(std::time(nullptr));
+        response->InsertHeader("Date", fmt::format("{:%a, %d %b %Y %H:%M:%S %Z}", tm));
+        if (req->RequestUrl() == "/plaintext") {
+          response->MutableBody() = "Hello, World!";
+        } else if (req->RequestUrl() == "/json") {
+          response->InsertHeader("Content-Type", "application/json");
+          response->MutableBody() = "{\"message\":\"Hello, World!\"}";
```
Comment (nbrady-techempower) [inline frameworks/C++/ltio/http_benchmark.cc:56]: For the JSON test, you must instantiate an object on each request.

See examples:

https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/C%2B%2B/cppcms/src/test_simple.cpp#L17

https://github.com/TechEmpower/FrameworkBenchmarks/blob/master/frameworks/C%2B%2B/drogon/drogon_benchmark/controllers/JsonCtrl.cc#L7

Comment (echoface) [reply frameworks/C++/ltio/http_benchmark.cc]: done. thanks again.

**Comment by echoface** [comment]
a latest update commit has done, please help review and merge. @nbrady-techempower , thanks.

**Pull request merged**